Cannot install gopass with go get

### Summary
I don't know why but when i run `go get github.com/gopasspw/gopass`
It return this error and then stop install:
```
# github.com/gopasspw/gopass/pkg/completion/zsh
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:25:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.BoolFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:27:8: undefined: cli.BoolTFlag
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:29:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Float64Flag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:31:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.GenericFlag (IsSet method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:33:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64Flag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:35:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64SliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:37:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:39:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntSliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:41:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:43:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringSliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:43:3: too many errors
# github.com/gopasspw/gopass/pkg/completion/fish
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:42:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.BoolFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:44:8: undefined: cli.BoolTFlag
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:46:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Float64Flag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:48:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.GenericFlag (IsSet method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:50:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64Flag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:52:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64SliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:54:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:56:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntSliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:58:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:60:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringSliceFlag (Apply method has pointer receiver)
go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:60:3: too many errors
```

### Environment
- OS: Linux parrot 5.3.0-2parrot2-amd64 #1 SMP Parrot 5.3.9-3parrot2 (2019-11-22) x86_64 GNU/Linux
- Installation method: go get

--- a/go.mod
+++ b/go.mod
@@ -4,12 +4,13 @@ go 1.12
 
 require (
 	github.com/alecthomas/binary v0.0.0-20190922233330-fb1b1d9c299c
+	github.com/alexflint/go-arg v1.3.0 // indirect
 	github.com/atotto/clipboard v0.1.2
 	github.com/blang/semver v0.0.0-20190414182527-1a9109f8c4a1
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/dominikschulz/github-releases v0.0.0-20171016132906-7635b43b2447
+	github.com/dominikschulz/github-releases v0.0.2
 	github.com/doronbehar/gocui v0.4.2
 	github.com/fatih/color v1.9.0
 	github.com/gdamore/tcell v1.3.0

